assets/app/views/directives/osc-key-values.html

```diff
@@ -23,7 +25,15 @@
                  spellcheck="false"
                  osc-input-validator="value">
         </div>
-        <button class="btn btn-default" ng-click="addEntry()" ng-disabled="form.key.$error.oscKeyValid || form.value.$error.oscValueValid">Add</button>
+        <button
+          class="btn btn-default"
+          ng-click="addEntry()"
+          ng-disabled="form.$invalid || form.$pristine">
```

Comment:
Pristine will be false if I set a value and then delete it. It's probably better to test if a value is set in the model.